packages/permission-controller/package.json

```diff
@@ -35,7 +35,7 @@
     "@metamask/controller-utils": "^5.0.2",
     "@metamask/json-rpc-engine": "^7.1.1",
     "@metamask/rpc-errors": "^6.1.0",
-    "@metamask/utils": "^8.2.0",
+    "@metamask/utils": "^8.1.0",
```

Comment:
Why is this downgrading these packages? 🤔

Reply:
Thanks for catching this! https://github.com/MetaMask/core/pull/1895#discussion_r1381269939